src/js/components/Header/AccountMenu/AccountMenu.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Button,
+  Flex,
+  Modal,
+  ModalOverlay,
+  Text,
+  Tooltip,
+} from "@chakra-ui/react";
+import * as React from "react";
+import useGlobalSettings from "../../../GlobalSettings/useGlobalSettings";
+import useAccount from "../../AccountProvider/useAccount";
+import AccountContent from "./ModalContents/AccountContent";
+import ConnectWalletContent from "./ModalContents/ConnectWalletContent";
+
+function AccountMenu() {
+  const { account } = useAccount();
+  const { isPhoneHardware } = useGlobalSettings();
+  const [isAccountModalOpen, setIsAccountModalOpen] =
+    React.useState<boolean>(false);
+  const [isConnectModalOpen, setIsConnectModalOpen] =
+    React.useState<boolean>(false);
+
+  const onCloseModal = () => {
+    setIsAccountModalOpen(false);
+    setIsConnectModalOpen(false);
+  };
+
+  const onAccountModalOpen = () => {
+    setIsAccountModalOpen(true);
+  };
+
+  const onConnectModalOpen = () => {
+    setIsConnectModalOpen(true);
+  };
```

Comment:
For these cases Chakra-UI has nice hook `useDisclosure`
https://chakra-ui.com/docs/hooks/use-disclosure

Reply:
Done